runestone/assess/js/mchoice.js

```diff
@@ -428,10 +428,10 @@ MultipleChoice.prototype.renderMCMAFeedBack = function () {
     var feedbackText = this.feedbackString;
 
     if (this.correct) {
-        $(this.feedBackDiv).html('Correct.<ol type="A">' + feedbackText + "</ul>");
-        $(this.feedBackDiv).attr("class", "alert alert-success");
+        $(this.feedBackDiv).html('✔️ <ol type="A">' + feedbackText + "</ul>");
+        $(this.feedBackDiv).attr("class", "alert alert-info");
     } else {
-        $(this.feedBackDiv).html("Incorrect.    " + "You gave " + numGiven +
+        $(this.feedBackDiv).html("Not quite.    " + "You gave " + numGiven +
```

Comment:
A minor nitpick -- why not have an X here to match the checkmark above? That seems more consistent to me. Other than that, I'm fine with this.